src/test/java/BitcoinTest.java

```diff
@@ -8,6 +12,9 @@ public void BitcoinTest() {
 
         BitcoinService bitcoinService = new BitcoinService();
         bitcoinService.getListOfAverages();
+        BlockchainService blockchainService = new BlockchainService();
+        blockchainService.getListOfY();
+        Assert.assertEquals(bitcoinTestComparator(bitcoinService.getListOfAverages(), blockchainService.getListOfY()), true, "Test failed!");
```

Comment:
1.static import for Assert
2.assertEquals(something, true) -> assertTrue(something)
3.Test failed is a horrible error message